### ISSUE TITLE
[Question] Are you going to add shared resources?

### ISSUE THREAD

**Issue by Librixf** [doitsujin/dxvk#3831]
Are you going to add shared resources? FiveM, RageMP and I think also EAC depend on them and I saw https://github.com/doitsujin/dxvk/issues/1722 , and I would like to check the situation 3 years later.

Sorry if I'm annoying with this question.

**Comment by Blisto91** [comment]
Shared resources have been mostly supported for a while now. Is there a game you are having issues with?

**Comment by K0bin** [comment]
Fwiw, no, EAC is not blocked by shared resources.

**Comment by Librixf** [comment]
> Shared resources have been mostly supported for a while now. Is there a game you are having issues with?

To tell you the truth - it's FiveM. I asked in a Linux chat room and was told that dxvk doesn't support some dll files, but then I saw this thread https://github.com/doitsujin/dxvk/issues/1722 and thought that it's just the shared resources that are not supported.

**Comment by Blisto91** [comment]
Of the top of my mind I am not aware of any issues with dxvk and FiveM on Linux right now. Haven't checked myself though.
If there are problems please tell them to open an issue.

**Comment by Librixf** [comment]
> If there are problems please tell them to open an issue.

Very sorry for the long wait, I'll attach the logs. I tried to run through PortProton and version 8.0-4
[PortProton.log](https://github.com/doitsujin/dxvk/files/14232597/PortProton.log)

**Comment by Blisto91** [comment]
I think it's better to create a new issue with full information about what the issue is.
I am not seeing any dxvk output (or much at all) in your attached log.

**Comment by Blisto91** [comment]
> 0168:err:combase:RoGetActivationFactory Failed to find library for L"Windows.UI.Xaml.Hosting.WindowsXamlManager"

This is likely the Wine error causing you issues. Which i see myself when trying to get FiveM installed.
It will hit it when the Wine prefix it set to use Windows 10 or 11, but using 8.1 or will make the installer complain about the Windows version not being supported

**Comment by Librixf** [comment]
I'm sorry, I sent you the wrong log. It's on FiveM.

**Comment by Librixf** [comment]
[PortProton.log](https://github.com/doitsujin/dxvk/files/14233200/PortProton.log)
This is the log for RAGE MP, I may be wrong, but there are Vulkan related errors and dxvk may also be to blame.

**Comment by Librixf** [comment]
> This is likely the Wine error causing you issues. Which i see myself when trying to get FiveM installed.
> It will hit it when the Wine prefix it set to use Windows 10 or 11, but using 8.1 or will make the installer complain about the Windows version not being supported

Thank you for pointing that out.

**Comment by Librixf** [comment]
> Which i see myself when trying to get FiveM installed.

Did it work out in the end?

**Comment by Blisto91** [comment]
No there is a Wine issues preventing the installer from running when running it in a Windows 10 or 11 prefix. And didn't mess much with trying to get past the unsupported popup when trying 8.1 which support ended for here last month.

**Comment by ZimZom** [comment]
Are there any new Infos about the Shared Resources and FiveM Problems? Is there today any chances?

**Comment by K0bin** [comment]
Shared Resources are supported on Linux in Proton. No idea whether GTA V mods work.

**Comment by Blisto91** [comment]
As said above Shared resources have been supported for a couple of years now with Proton. FiveM doesn't work on Linux because of some UWP issue last i checked. Not aware of any dxvk problems

**Comment by ZimZom** [comment]
Okay. Is it right that if I try to play FiveM on Linux and I try to connect on a Server the Server must deactivate the svadhesive Option? And I can start FiveM without any Problems?

**Comment by K0bin** [comment]
No idea. I don't like GTA.

**Comment by JustOptimize** [comment]
> As said above Shared Resources have been supported for a couple of years now with Proton. FiveM doesn't work on Linux with Proton because of some UWP issue last i checked. Not aware of any dxvk problems

Can confirm that FiveM doesn't launch because it can't find "Windows.UI.Xaml.Hosting.WindowsXamlManager".
If anyone knows how to bypass the windows 7 lock or fix the win 10 error let us know
